src/main/java/ch/heigvd/amt/mvcProject/domain/question/Question.java

```diff
@@ -0,0 +1,42 @@
+package ch.heigvd.amt.mvcProject.domain.question;
+
+import ch.heigvd.amt.mvcProject.domain.IEntity;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+import java.util.List;
+
+@Data
+@Builder(toBuilder = true)
+public class Question implements IEntity {
+
+    @Setter(AccessLevel.NONE)
+    private QuestionId id;
+
+    private String title;
+
+    private String description;
+
+    private List<String> tags;
+
+    /**
+     * Override the builder generated by Lombok
+     */
+    public static class QuestionBuilder {
```

Comment:
I think you need to initialize the tags list if the builder didn't pass tags, you could have a nullptr otherwise